internal/rego/rego.go

```diff
@@ -174,7 +175,7 @@ func (r Rego) PolicyID() string {
 // found in the header comment of the rego file.
 func (r Rego) Description() string {
 	var description string
-	for _, comment := range r.comments {
+	for _, comment := range r.headerComments {
 
 		// When the  token appears, we consider this point to
```

Comment:
I believe we no longer want `@kinds` to end the header processing, right?

Reply:
Ah yes, good catch!